component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/AuthPages.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.casque.authenticator.authenticator;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.casque.authenticator.authenticator.radius.RadiusPacket;
+import org.wso2.carbon.identity.casque.authenticator.constants.CasqueAuthenticatorConstants;
+import org.wso2.carbon.identity.casque.authenticator.exception.CasqueException;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Serializable;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This Class returns the CASQUE Challenge HTML Page
+ */
+class AuthPages implements Serializable {
+
+    private static final long serialVersionUID = 4341535155455223655L;
+    private static final Log log = LogFactory.getLog(RadiusPacket.class);
+
+    /**
+     * Add the text/html content type to the response and forward to returnResponse()
+     *
+     * @param response Http servlet response
+     * @param data     the content for the HTTP response
+     * @throws IOException :
```

Comment:
Missing description. Fix it in the other places aswell

Reply:
fix with - https://github.com/wso2-extensions/identity-outbound-auth-casque-snr/pull/8